.cookiecutter/cookiecutter.json

```diff
@@ -1,30 +1,31 @@
 {
   "template": "https://github.com/hypothesis/cookiecutters",
   "checkout": null,
-  "directory": "pyramid-app",
-  "ignore": [],
+  "directory": "pyapp",
+  "ignore": ["conf/supervisord*.conf", "report/__main__.py", "report/app.py", "report/core.py", "tests/unit/report/core_test.py"],
```

Comment:
[Discussed in Slack](https://hypothes-is.slack.com/archives/C1MA4E9B9/p1668440590863119): I don't think Supervisor currently makes sense for the `report` repo. The `supervisord[-dev].conf` files that're currently in this repo don't actually run any processes other than the `logger` process. The `supervisord.conf` file is unused: the `Dockerfile` doesn't run it. `supervisord-dev.conf` is used by `make dev` on `main` but `make dev` is itself pointless currently: it just runs the logger process.